app/templates/settings.hbs

```diff
@@ -1,139 +1,153 @@
-      {{/if}}
-    {{/if}}
-
-    {{#unless model.hasPushAccess}}
-      <p>You don't have sufficient permissons to add or remove ssh keys on this repository.</p>
-    {{/unless}}
-
-  </section>
-{{/if}}
+	<section class="settings-section">
+		<h2 class="small-title">General</h2>
+		<ul class="settings-list--columns">
+			<li>
+				{{settings-switch active=model.settings.build_pushes repo=repo description="Build pushed branches" key="build_pushes"}}
+				<a href="https://docs.travis-ci.com/user/web-ui/#build-pushed-branches" title="about branch updates"
+					class="settings-tooltip">
+					{{ember-tooltip text='Read more about branch updates'}}
+					{{svg-jar 'icon-help' class="icon-help"}}</a>
+			</li>
+			<li>
+				{{limit-concurrent-builds value=model.settings.maximum_number_of_builds enabled=concurrentBuildsLimit repo=repo}}
+			</li>
+			<li>
+				{{settings-switch active=model.settings.build_pull_requests repo=repo description="Build pushed pull requests" key="build_pull_requests"}}
+				<a href="https://docs.travis-ci.com/user/web-ui/#build-pushed-pull-requests" title="about pull request updates"
+					class="settings-tooltip">
+					{{ember-tooltip text='Read more about pull request updates'}}
+					{{svg-jar 'icon-help' class="icon-help"}}</a>
+			</li>
+		</ul>
+	</section>
+
+	{{#if showAutoCancellationSwitches}}
+	<section class="settings-section auto-cancellation">
+		<h2 class="small-title display-inline">Auto Cancellation</h2>
+
+		<p>Auto Cancellation allows you to only run builds for the latest commits in the queue. This setting can be
+			applied to builds for Branch builds and Pull Request builds separately. Builds will only be canceled if they
+			are waiting to run, allowing for any running jobs to finish.</p>
+
+		<ul class="settings-list--columns">
+			<li>
+				{{settings-switch active=model.settings.auto_cancel_pushes repo=repo description="Auto cancel branch builds" key="auto_cancel_pushes"}}
+			</li>
+			<li>
+				{{settings-switch active=model.settings.auto_cancel_pull_requests repo=repo description="Auto cancel pull request builds" key="auto_cancel_pull_requests"}}
+			</li>
+		</ul>
+	</section>
+	{{/if}}
+
+	{{#if showAllowConfigImportsSwitch}}
+	<section class="settings-section config-import">
+		<h2 class="small-title display-inline">Config Import</h2>
+
+		<ul class="settings-list--columns">
+			<li>
+				{{settings-switch active=model.settings.allow_config_imports repo=repo description="Allow importing config files from this repository" key="allow_config_imports"}}
+			</li>
+		</ul>
+	</section>
+	{{/if}}
+
+	<section class="settings-section">
+		<h2 class="small-title">Environment Variables</h2>
+
+		<p>Customize your build using environment variables. For secure tips on generating private keys
+			<a
+				href='https://docs.travis-ci.com/user/best-practices-security#recommendations-on-how-to-avoid-leaking-secrets-to-build-logs'>
+				read our documentation
+			</a>.</p>
```

Comment:
`a` looks like a one-liner as it has only one attribute, and they all should be nested inside `p`:
```
<p>
  Customize your build using environment variables. For secure tips on generating private keys
  <a href='https://docs.travis-ci.com/user/best-practices-security#recommendations-on-how-to-avoid-leaking-secrets-to-build-logs'>
    read our documentation
  </a>.
</p>
```